content/operate/kubernetes/reference/redis_enterprise_database_api.md

```diff
@@ -0,0 +1,1633 @@
+          <br/>
+            <i>Format</i>: int64<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b>redisEnterpriseCluster</b></td>
+        <td>string</td>
+        <td>
+          The Redis Enterprise Cluster Object this Resource is associated with<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b><a href="#statusreplicasourcestatusesindex">replicaSourceStatuses</a></b></td>
+        <td>[]object</td>
+        <td>
+          ReplicaSource statuses<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b>shardStatuses</b></td>
+        <td>map[string]integer</td>
+        <td>
+          Aggregated statuses of shards<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b>specStatus</b></td>
+        <td>string</td>
+        <td>
+          Whether the desired specification is valid<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b>status</b></td>
+        <td>string</td>
+        <td>
+          The status of the database<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b>version</b></td>
+        <td>string</td>
+        <td>
+          Database compatibility version<br/>
+        </td>
+        <td>false</td>
+      </tr></tbody>
+</table>
+
+
+### status.activeActive
+<sup><sup>[↩ Parent](#status)</sup></sup>
+
+Connection/ association to the Active-Active database.
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b>name</b></td>
+        <td>string</td>
+        <td>
+          The the corresponding Active-Active database name, Redis Enterprise Active Active Database custom resource name, this Resource is associated with. In case this resource is created manually at the active active database creation this field must be filled via the user, otherwise, the operator will assign this field automatically. Note: this feature is currently unsupported.<br/>
+        </td>
+        <td>true</td>
+      </tr><tr>
+        <td><b>participatingClusterName</b></td>
+        <td>string</td>
+        <td>
+          The corresponding participating cluster name, Redis Enterprise Remote Cluster custom resource name, in the Active-Active database, In case this resource is created manually at the active active database creation this field must be filled via the user, otherwise, the operator will assign this field automatically. Note: this feature is currently unsupported.<br/>
```

Comment:
```suggestion
          The corresponding participating cluster name, or Redis Enterprise Remote Cluster custom resource name, in the Active-Active database. If this resource is created manually at the Active-Active database creation, this field must be filled via the user, otherwise, the operator will assign this field automatically. Note: this feature is currently unsupported.<br/>
```